hypha/apply/determinations/forms.py

```diff
@@ -403,3 +404,18 @@ def __init__(self, *args, submissions, initial={}, **kwargs):
         self.fields['outcome'].widget = forms.HiddenInput()
 
         self.fields = self.apply_form_settings('proposal', self.fields)
+
+
+class UpdateDeterminationForm(forms.ModelForm):
+    message = RichTextField(label='Determination message')
+
+    class Meta:
+        model = Determination
+        fields = ['outcome', 'message']
```

Comment:
Remove outcome and add send_notice.

Reply:
They also want all the other text fields.

Reply:
oh okay, Need to add more changes in that case, as form fields are different for 4 workflows.

Reply:
I have used existing form with few modifications.